tests/sanity/ignore-2.9.txt

```diff
@@ -36,91 +36,18 @@ plugins/modules/cloud/dimensiondata/dimensiondata_vlan.py validate-modules:doc-c
 plugins/modules/cloud/dimensiondata/dimensiondata_vlan.py validate-modules:doc-missing-type
 plugins/modules/cloud/dimensiondata/dimensiondata_vlan.py validate-modules:parameter-type-not-in-doc
 plugins/modules/cloud/docker/docker_container.py use-argspec-type-path # uses colon-separated paths, can't use type=path
-plugins/modules/cloud/google/gc_storage.py validate-modules:doc-choices-do-not-match-spec
-plugins/modules/cloud/google/gc_storage.py validate-modules:doc-default-does-not-match-spec
-plugins/modules/cloud/google/gc_storage.py validate-modules:doc-missing-type
-plugins/modules/cloud/google/gc_storage.py validate-modules:parameter-type-not-in-doc
-plugins/modules/cloud/google/gc_storage.py validate-modules:undocumented-parameter
-plugins/modules/cloud/google/gcdns_record.py validate-modules:deprecation-mismatch
```

Comment:
You need to keep the deprecation-mismatch lines in ignore-2.9.txt.

Reply:
how can I force ansible-test to run against 2.9 locally? my regular cmd line is:
```
$ ansible-test sanity --docker default -v plugins/modules/cloud/google/gc_storage.py
```
and from what I can tell out of the output, it runs against ansible 2.10 and I do not see any obvious parameter in the `--help` output about that.

Reply:
You need to install Ansible 2.9 for that (using a venv would be a good idea). ansible-test comes with ansible(-base), so there's no verison selector.